index.html

```diff
@@ -287,14 +597,65 @@ <h2>Grammar Extensions</h2>
             resolves to a single property/value relationship.</li>
         </ul>
       </li>
+      <li>A <a>node object</a> or a <a>value object</a> MAY contain an `@annotation` property (or an alias) referencing
+        zero or more <a>annotation objects</a>.</li>
+      <li>A <a>node object</a> which is not the value of a non-keyword property,
+        or which is an value of a <a>list object</a>,
+        MUST NOT contain an `@annotation` property.</li>
```

Comment:
* by "value of a property", we also include "element of an array that is a value of a non-keyword property". Is it clearly specified elsewhere, or should it be made explicit?

* actually, the values of `@none` in a language map would be reasonable candidates for having an `@annotation`...

Reply:
By "value of a property", it is certainly reasonable that the property may have an array of values, one of which is annotated. I'm not sure this needs to be stated specifically, unless you think it unclear.

The data model is really that of the expanded representation, so the maps disappear. Language maps can't really be used, though, as their values are restricted to be strings; they would need to be value objects, which is currently not allowed. It seems like a reasonable limitation.

Reply:
Re "value of a property", I think it is ok, but I just wanted to check.
Re language maps, you are right, I got carried away.